specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/examples/sshPublicKeyExamples/SshPublicKey_GenerateKeyPair_EncryptionWithEd25519.json

```diff
@@ -2,17 +2,17 @@
   "parameters": {
     "subscriptionId": "{subscription-id}",
```

Comment:
should be updating 2024-07-01 version folder !

Reply:
also please update title to reflect your change

Reply:
done!